addons/packages/cartographer/0.2.2/README.md

```diff
@@ -0,0 +1,120 @@
+# Package for Cartographer
```

Comment:
this is straight from the `vmware-tanzu/package-for-cartographer` repository where the contents have been written really focusing on that repository itself and not `community-edition` - is there a specific way that we should be writing this?

Reply:
went ahead with mimicking harbor and kpack